algebird-core/src/main/scala/com/twitter/algebird/BloomFilter.scala

```diff
@@ -369,12 +369,41 @@ sealed abstract class BF[A] extends java.io.Serializable {
   // Estimates the cardinality of the set of elements that have been
   // inserted into the Bloom Filter.
   def size: Approximate[Long]
+
+  def toBitSet: BitSet
+
+  /**
+   * Compute the Hamming distance between the two Bloom filters
+   * `a` and `b`. The distance is defined as the number of bits that
+   * need to change to in order to transform one filter into the other.
+   */
+  def hammingDistance(that: BF[A]): Int = {
+    assert((this.width == that.width) && (this.numHashes == that.numHashes),
```

Comment:
I'd be happy to move this anywhere you think it should go. I started out keeping it in the `BloomFilter` and I can place it back in there, or move it to the `BF` companion object if you prefer that.